pandas/tseries/offsets.py

```diff
@@ -1121,7 +1125,7 @@ class SemiMonthOffset(DateOffset):
     _default_day_of_month = 15
     _min_day_of_month = 2
 
-    def __init__(self, n=1, day_of_month=None, normalize=False, **kwds):
+    def __init__(self, n=1, normalize=False, day_of_month=None):
```

Comment:
is this consistent with other freq signatures?

Reply:
All the other signatures are `(self, n=1, normalize=False, ...`.  In the status quo this is an outlier.

Reply:
yes that's fine. see my comment below though.

Reply:
this is fine, in an ideal world we would change it but can't now